src/frontend/public/locales/es/translation.json

```diff
@@ -1,5 +1,70 @@
 {
     "discover": "Descubrir",
     "compare": "Comparar",
-    "logout": "Cerrar sesión"
+    "neighborhoodDetail": {
+        "addToFavorites": "Add to favorites",
+        "moveCountText": "¡EXAMPLE {{count}} titulares de vales ya se mudaron aquí!",
```

Comment:
I added just a couple translated examples (all prefaced with EXAMPLE) just to confirm translations work when switching languages